index.js

```diff
@@ -14,6 +14,7 @@ const router = require('./lib/router');
 const health = require('./lib/health');
 const serveStatic = require('./lib/serve-static');
 const gaTagSetup = require('./lib/ga-tag');
+const deIndexer = require('./lib/deindex');
```

Comment:
similar to above, is this needed for the postcode lookup work?

Reply:
Removed this now.